crates/bins/src/bin/datadog-static-analyzer-git-hook.rs

```diff
@@ -124,6 +128,13 @@ fn main() -> Result<()> {
         "484df034272a3f63c1796ef44f83a790d7729590",
     );
 
+    opts.optopt(
+        "",
+        "output",
+        "file that contains all findings from the Git hooks (SARIF output)",
```

Comment:
it's nice that we're describing the file, but it doesn't explain what the relationship between the file and the cli arg is (e.g. in the main cli we say "output file name", which is clearer)

```suggestion
        "output file name to write all findings from the Git hooks to",
```